sample/src/androidTest/kotlin/com/agoda/sample/ChipGroupTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.agoda.sample
+
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import androidx.test.rule.ActivityTestRule
+import com.agoda.kakao.screen.Screen.Companion.idle
+import com.agoda.kakao.screen.Screen.Companion.onScreen
+import com.agoda.sample.screen.ChipGroupActivityScreen
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+class ChipGroupTest {
+    @Rule
+    @JvmField
+    val rule = ActivityTestRule(ChipGroupActivity::class.java)
+
+    @Test
+    fun testChipGroupInteraction() {
+        onScreen<ChipGroupActivityScreen> {
+            chipGroup {
+                selectChip("Chip2")
+
+                idle(5000)
```

Comment:
Please do not idle for that long. 500-1000 ms is more than enough. But actually, there should not be any idle at all.

Reply:
Whoa. I totally forgot to about that line. Was using it for testing. Definitely not supposed to be there :sweat_smile: